### PR TITLE
Added a fix for BSC#1226285.

### PR DESCRIPTION
**Pull request by lvicoun**
### PR creator: Description

Adding how to workaround installation problem.

### PR creator: Are there any relevant issues/feature requests?

* bsc#1226285

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [ ] SLE 15 next/openSUSE Leap next *(current `main`, no backport necessary)*
  - [ ] SLE 15 SP5/openSUSE Leap 15.5
  - [ ] SLE 15 SP4/openSUSE Leap 15.4
  - [ ] SLE 15 SP3/openSUSE Leap 15.3
  - [ ] SLE 15 SP2/openSUSE Leap 15.2

- SLE 12
  - [ ] SLE 12 SP5

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

File: xml/aarch64_inst_problem.xml [hunk 1]
```diff
@@ -104,6 +104,36 @@
...
+  <sect3 xml:id="sec-install-ipmi">
+    <title>Remote Installation using an IPMI console</title>
+    <para>
+      When installing remotely using an IMPI console, you may face problems with the installation.
+      To fix the problems and install &productnameshort;, proceed as follows:
+    </para>
+    <procedure>
+      <step>
+        <para>
+          Press <keycap>e</keycap> to enter the GRUB menu.
```
[the remark this answers is not in the document]
Comment (denknorr) [reply xml/aarch64_inst_problem.xml]: i saw your change, this is fine!

**Review by denknorr — APPROVED** [review]

File: xml/aarch64_inst_problem.xml [hunk 2]
```diff
@@ -104,6 +104,36 @@
     </para>
    </listitem>
   </itemizedlist>
+  <sect3 xml:id="sec-install-ipmi">
+    <title>Remote Installation using an IPMI console</title>
```
Comment (dariavladykina) [inline xml/aarch64_inst_problem.xml:108]:
```suggestion
    <title>Remote installation using an IPMI console</title>
```

File: xml/aarch64_inst_problem.xml [hunk 3]
```diff
@@ -104,6 +104,36 @@
...
+    <para>
+      When installing remotely using an IMPI console, you may face problems with the installation.
```
Comment (dariavladykina) [inline xml/aarch64_inst_problem.xml:110]:
```suggestion
      When installing remotely using an IPMI console, you may face problems with the installation.
```

File: xml/aarch64_inst_problem.xml [hunk 4]
```diff
@@ -104,6 +104,36 @@
...
+      To fix the problems and install &productnameshort;, proceed as follows:
+    </para>
+    <procedure>
+      <step>
+        <para>
+          Press <keycap>e</keycap> during start of the installer to enter the GRUB menu.
```
Comment (dariavladykina) [inline xml/aarch64_inst_problem.xml:116]:
```suggestion
          Press <keycap>e</keycap> during the start of the installer to enter the GRUB menu.
```

File: xml/aarch64_inst_problem.xml [hunk 5]
```diff
@@ -104,6 +104,36 @@
...
+      </step>
+      <step>
+        <para>      
+           Replace <literal>splash=silent</literal> with <literal>ConsoleDevice=/dev/tty1</literal>. 
+        </para>
+      </step>
+      <step>
+        <para>
+          Continue to boot with <keycombo><keycap function="control"/><keycap>x</keycap></keycombo> or <keycap>F10</keycap>. The installer
+          will start in the terminal on <keycombo><keycap function="alt"/><keycap>F1</keycap></keycombo>.
```
Comment (dariavladykina) [inline xml/aarch64_inst_problem.xml:127]:
```suggestion
          Continue to boot with <keycombo><keycap function="control"/><keycap>x</keycap></keycombo> or <keycap>F10</keycap>. Press <keycombo><keycap function="alt"/><keycap>F1</keycap></keycombo> to launch the installer in a terminal window.
```

**Review by dariavladykina — COMMENTED** [review]
Hi, please see some suggestions. Thanks!

**Pull request merged**